examples/charts/file/hover_span.py

```diff
@@ -10,7 +10,7 @@
 
 TOOLS='box_zoom,box_select,hover,crosshair,reset'
 
-TOOLTIPS = [ ("y", "$~y"), ("x", "$~x") ]
+TOOLTIPS = [ ("y", "$~y"), ("x", "$~x{0}") ]
```

Comment:
This might be considered personal preference and I don't have a problem rolling this back if others disagree with it.